### ISSUE TITLE
Map Filter - ANDROID 1.3.4 Orientation - Three buttons are overlapping the map on Android (Details, Time Filter, Saved & Search)

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9984]
As a mobile user, I expect all components on the page to reflow into a usable format on mobile, with no components cut off or obscured.

**Linked Work Item:**
#9493 

**Expected Result:**
When using the site on mobile, all components should be visible and should not overlap one another, should not be cut off at the edge of the page and should not require the user to scroll in two dimensions in order to see them. 

**Actual Result:**
Four buttons are overlapping the map on Android and not all of the buttons are visible or accessible. This will impact user's ability to access the map and to the buttons themselves.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/878075f7-b06c-4bd5-89be-bb09b32aa9d2)

**Comment by apeters** [comment]
closed by #10104

**Comment by imogenroberts** [comment]
RETEST: Textual buttons have been replaced with graphical buttons on mobile which are all clearly visible on mobile. @johnatawnclementawn just make sure that each of these has clear, concise alt-text (aria) so people using screen readers on mobile can tell what they are. 

**Evidence:**
![Map Filter - ANDROID 1 3 4 Orientation](https://github.com/archesproject/arches/assets/77161085/7e79946a-11b0-4dc9-9a8c-583c49e1d9aa)
